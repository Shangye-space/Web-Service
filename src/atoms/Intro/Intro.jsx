import React from 'react';

const Intro = () => {
    return (
        <>
            <div className="overlay"></div>
            <video
                playsInline="playsinline"
                autoPlay="autoplay"
                muted="muted"
                loop="loop"
            >
                <source src="/public/videos/bgv.mp4" type="video/mp4" />
            </video>
        </>
    );
};

export default Intro;
