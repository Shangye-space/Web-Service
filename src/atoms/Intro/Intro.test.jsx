import React from 'react';
import ReactDOM from 'react-dom';
import Intro from './Intro.jsx';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Intro />, div);
    ReactDOM.unmountComponentAtNode(div);
});

it('snapshot renders', () => {
    const component = renderer.create(<Intro />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
