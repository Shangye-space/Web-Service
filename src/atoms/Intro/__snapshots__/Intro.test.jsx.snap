// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot renders 1`] = `
Array [
  <div
    className="overlay"
  />,
  <video
    autoPlay="autoplay"
    loop="loop"
    muted="muted"
    playsInline="playsinline"
  >
    <source
      src="/videos/bgv.mp4"
      type="video/mp4"
    />
  </video>,
]
`;
